### DISCUSSION TITLE
What is in it for Time Travel??!!!

### DISCUSSION THREAD

**Discussion by kishanrayanal** [community/community#17740] [GitHub Education]
Are our Dreams connected to parallel universes of us in different dimensions>>>!!!!

**Comment by suprstarrd** [comment]
We developers know a lot of programming languages. The brain is (usually) not one of them.

**Comment by samus-aran** [comment]
Hi @kishanrayanal 

Thanks for being a part of the GitHub Community! We're glad you're here at this time and place. ⏳ 

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on the philosophy of time travel. It's definitely possible that another GitHub user might have run into this same issue (doubt it) and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic. ⌛ 

Best of luck! 🍀
